Gamma_Analysis.py

```diff
@@ -206,8 +217,8 @@ def peak_measurement(M, energy, sub_regions='both'):
     if len(compton_region) < 2:
         compton_region_uncertainty = (compton_region[0])**0.5
     else:
-        compton_region_uncertainty = (compton_region[0] +
-                                      compton_region[1])**0.5
+        compton_region_uncertainty = (2 * (compton_region[0] +
+                                           compton_region[1]))**0.5
```

Comment:
I think the 2 goes in the denominator here, right? And outside the square root (my bad). Because the uncertainty of the total Compton counts is sqrt(left counts + right counts), and the uncertainty of total / 2 is just uncertainty / 2.